media/css/etc/firefox/retention/thank-you.scss

```diff
@@ -0,0 +1,161 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../../pebbles/includes/lib';
+@import '../../../pebbles/components/newsletter';
+
+body {
+    border-top: 0;
+}
+
+html {
+    font-size: 100%;
+}
+
+@font-face {
+    font-family: antonioregular;
+    src: url('/media/fonts/fx-lifestyle/antonio-regular-webfont.woff2') format('woff2'),
+         url('/media/fonts/fx-lifestyle/antonio-regular-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: antoniobold;
+    src: url('/media/fonts/fx-lifestyle/antonio-bold-webfont.woff2') format('woff2'),
+         url('/media/fonts/fx-lifestyle/antonio-bold-webfont.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: normal;
+    font-style: normal;
+    src: url('/media/fonts/opensans-regular.woff2') format('woff2'),
+         url('/media/fonts/opensans-regular.woff') format('woff');
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: bold;
+    font-style: normal;
+    src: url('/media/fonts/opensans-semibold.woff2') format('woff2'),
+         url('/media/fonts/opensans-semibold.woff') format('woff');
+}
```

Comment:
I couldn't get semibold to work with any default, and after trying again now, still not sure how I can get semibold to work, and it was specifically requested during design feedback that I use it.

I don't see semi bold defined anywhere, and looking at the code, it looks like it's added on a case by case basis.

I am going to remove the regular non bold one though, as that's already there. I'm also going to be more specific with semibold and set it as 600, instead of overwriting the existing bold.